Application/EdFi.Ods.Api/Security/Authorization/EntityAuthorization/EntityAuthorizationSqlBuilder.cs

```diff
@@ -0,0 +1,124 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Linq;
+using System.Text;
+using EdFi.Common.Utils.Extensions;
+using EdFi.Ods.Api.Security.Authorization.Repositories;
+using EdFi.Ods.Api.Security.AuthorizationStrategies.Relationships.Filters;
+using EdFi.Ods.Common.Security.Authorization;
+
+namespace EdFi.Ods.Api.Security.Authorization.EntityAuthorization;
+
+public interface IEntityAuthorizationSqlBuilder
+{
+    string BuildExistenceCheckSql(AuthorizationStrategyFilterResults[] resultsWithPendingExistenceChecks);
+}
+
+public class EntityAuthorizationSqlBuilder : IEntityAuthorizationSqlBuilder
+{
+    public string BuildExistenceCheckSql(AuthorizationStrategyFilterResults[] resultsWithPendingExistenceChecks)
```

Comment:
Should unit tests be added to explicitly cover this implementation, or is its functionality sufficiently covered by other unit/integration tests?